src/components/JoinList.js

```diff
@@ -1,36 +1,45 @@
-import React, { useEffect } from 'react';
+import React from 'react';
```

Comment:
I'm finally just learning hooks myself this weekend! All of the examples from the official docs have this import statement including `useState` so you can reference it directly within the component. This isn't a blocking comment because what you have does work, but it does seem more canonical to do it like this:

    import React, { useState } from 'react';